.github/ISSUE_TEMPLATE/4.docs-feedback.yml

```
@@ -0,0 +1,52 @@
+name: Docs feedback
+description: Improve documentation about MUI Core.
+labels: ['status: needs triage', 'docs']
```

I almost feel like it would be better without this tag, until we confirm that it's actually an issue with the docs, and not the feature itself.

```suggestion
labels: ['status: needs triage']
```